src/app/lib/streamer.js

```diff
@@ -1,501 +1,491 @@
-(function (App) {
+(function (App, WebTorrent) {
```

Comment:
it looks like you should just import webtorrent in that file, or pass it via App. we have enough global variables we don't want more =/